package.json

```diff
@@ -43,6 +43,7 @@
     "node-fetch": "2.7.0",
     "quicktype-core": "23.0.170",
     "type-fest": "4.26.1",
+    "undici": "^6.19.8",
```

Comment:
should use exact versions--but also would recommend we try using native-fetch first (same library) and bump the base required version to Node 22 for at least this feature in the docs

Reply:
okay will try that.

Reply:
The real struggle that I am running into with this is that native fetch does not have an agent. I am making an `https.globalAgent` but it is not respecting it. Still trying to work around this.